health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/icons/NewWindow.js

```
@@ -0,0 +1,18 @@
+import React from "react";
+
+export const NewWindow = () => {
+    return (
+      <svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 40 40" fill="none">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)